thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/job/JobScheduler.java

```diff
@@ -0,0 +1,20 @@
+package com.linkedin.thirdeye.anomaly.job;
+
+import java.util.List;
+
+import org.quartz.SchedulerException;
+
+public interface JobScheduler {
+
+  public List<String> getActiveJobs() throws SchedulerException;
+
+  public void start() throws SchedulerException;
+
+  public void stop() throws SchedulerException;
+
+  public void start(Long id) throws SchedulerException;
```

Comment:
start() / stop() are methods for JobSch.
if start(id) / stop(id) are meant for start/stopping  jobs, please rename these to start/stopJob(Long jobId)